### ISSUE TITLE
Kubeflow install on EKS is no longer possible due to open aws-efs-csi-driver bug

### ISSUE THREAD

**Issue by zoltan-fedor** [kubeflow/kubeflow#6150]
Due to the following bug of the aws-efs-csi-driver
https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/300

Kubeflow is NO longer compatible with AWS EKS.
This is because Kubeflow requires MySQL (or other) databases, which require persistent storage, which on AWS EKS means the usage of EFS, but the current way to use EFS on EKS is vait the aws-efs-csi-driver, which has the above mentioned bug.

End result that Kubeflow is no longer deployable on EKS.

**Comment by DavidSpek** [comment]
You don't need EFS for persistence. EKS comes with a default storage class that uses EBS, which will work fine and is also a more logical choice that EFS for databases.
/close

**Comment by zoltan-fedor** [comment]
Yes, that is what I ended up using too (RDS instead of on cluster MySQL). :-)
